src/Compilers/CSharp/Test/Symbol/Symbols/MissingSpecialMember.cs

```diff
@@ -572,6 +572,233 @@ public void AllWellKnownTypes()
+            Assert.True(WellKnownType.System_Xml_Linq_XCData <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Xml_Linq_XComment <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Xml_Linq_XContainer <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Xml_Linq_XDeclaration <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Xml_Linq_XDocument <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Xml_Linq_XElement <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Xml_Linq_XName <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Xml_Linq_XNamespace <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Xml_Linq_XObject <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Xml_Linq_XProcessingInstruction <= WellKnownType.CSharp7Sentinel);
+
+            Assert.True(WellKnownType.System_Security_UnverifiableCodeAttribute <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Security_Permissions_SecurityAction <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Security_Permissions_SecurityAttribute <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Security_Permissions_SecurityPermissionAttribute <= WellKnownType.CSharp7Sentinel);
+
+            Assert.True(WellKnownType.System_NotSupportedException <= WellKnownType.CSharp7Sentinel);
+
+            Assert.True(WellKnownType.System_Runtime_CompilerServices_ICriticalNotifyCompletion <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Runtime_CompilerServices_IAsyncStateMachine <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Runtime_CompilerServices_AsyncVoidMethodBuilder <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Runtime_CompilerServices_AsyncTaskMethodBuilder <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Runtime_CompilerServices_AsyncTaskMethodBuilder_T <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Runtime_CompilerServices_AsyncStateMachineAttribute <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Runtime_CompilerServices_IteratorStateMachineAttribute <= WellKnownType.CSharp7Sentinel);
+
+            Assert.True(WellKnownType.System_Windows_Forms_Form <= WellKnownType.CSharp7Sentinel);
+            Assert.True(WellKnownType.System_Windows_Forms_Application <= WellKnownType.CSharp7Sentinel);
+
+            Assert.True(WellKnownType.System_Environment <= WellKnownType.CSharp7Sentinel);
+
+            Assert.True(WellKnownType.System_Runtime_GCLatencyMode <= WellKnownType.CSharp7Sentinel);
```

Comment:
Are all members in WellKnownType tightly packed? Consider asserting actual difference between First and CSharp7Sentinel in order to make sure that none of the new types are inserted into the range.